packages/core/src/client.ts

```diff
@@ -328,8 +332,4 @@ export class Client {
     const operation = this.createRequestOperation('mutation', query, opts);
     return this.executeRequestOperation(operation);
   };
-
-  public get debugTarget() {
```

Comment:
Doesn't removing this mean exchanges could rewrite the debugTarget?

Reply:
They can always do that if they're not using TypeScript, this class is transpiled to a function either way

Reply:
We could set it as non-writeable, but I mean if we wanted to do that for every property and method, then we’d be doing this a lot 😅

I wouldn’t be too worried about this. When an exchange reassigns this they’ve already broken the API’s contract 🤷‍♀️